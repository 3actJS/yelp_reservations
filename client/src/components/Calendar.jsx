import React from 'react';

const Calendar = (props) => {
  return (
    <div id='calendar'></div>
  )
}

export default Calendar;