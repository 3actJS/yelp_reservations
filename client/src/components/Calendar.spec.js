/* eslint-disable no-undef */
/* eslint-disable no-unused-vars */
import React from 'react';
import { shallow } from 'enzyme';
import Calendar from './Calendar.jsx';

describe('Calendar', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Calendar debug />);

    expect(component).toMatchSnapshot();
  });
});
