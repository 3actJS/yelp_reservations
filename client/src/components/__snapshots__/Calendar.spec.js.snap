// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar Wrapper should render correctly in "debug" mode 1`] = `
<div
  className="sc-AxjAm fnKcEQ"
/>
`;
